publish2cloud.py

```diff
@@ -333,3 +335,34 @@ def publish_to_cloud(config, chunknum, check_versioning=None):
             publish_to_remote_settings(config, section, chunknum)
         else:
             print('Skipping Remote Settings upload for %s' % section)
+
+
+def request_review():
+    if check_upload_config(
+            config, 'main', 'remote_settings_upload'
+        ) == False:
+        print("\n*** Remote Settings upload is not enabled for this run, no reviews are required ***\n")
+        return
+
+    rs_collection_url = REMOTE_SETTINGS_URL + \
+         REMOTE_SETTINGS_PATH.format(
+            bucket_name=REMOTE_SETTINGS_BUCKET,
+            collection_name=REMOTE_SETTINGS_COLLECTION)
+
+    # Check if we need to send in a request for review
+    rs_collection = requests.get(rs_collection_url, auth=REMOTE_SETTINGS_AUTH, timeout=10)
+
+    if rs_collection:
+        # If any data was published, we want to request review for it
```

Comment:
Can you elaborate on the meaning of each status in the comment? It would be easier for people to understand.

Reply:
Added a comment